src/pages/yllapito/projekti/[oid]/henkilot.tsx

```diff
@@ -38,36 +40,75 @@ const loadedProjektiValidationSchema = getProjektiValidationSchema([
   ProjektiTestType.PROJEKTI_IS_CREATED,
 ]);
 
-export default function Henkilot({ setRouteLabels }: PageProps): ReactElement {
+export default function HenkilotPage({ setRouteLabels }: PageProps): ReactElement {
+  const { data: projekti, error: projektiLoadError, mutate: mutateProjekti } = useProjekti({ revalidateOnMount: true });
```

Comment:
yhtenäistä nimeäminen: mutateProjekti -> reloadProjekti.
Varmaan helpompi näin päin, kun reload on useammin.